Source/SPPreferencesUpgrade.m

```diff
@@ -80,11 +80,11 @@ void SPApplyRevisionChanges(void)
 	[prefs setObject:@YES forKey:@"doBundleUpdate"];
 
 	
-//	// FThis is how you add release notes and run specific migration steps
-//	if (recordedVersionNumber < 3922) {
-//		[importantUpdateNotes addObject:NSLocalizedString(@"The Custom Query \"Run\" and \"Run All\" button positions and their shortcuts have been swapped.", @"Short important release note for swap of custom query buttons")];
-//		[importantUpdateNotes addObject:NSLocalizedString(@"We've changed Sequel Ace's digital signature for GateKeeper compatibility; you'll have to allow access to your passwords again.", @"Short important release note for why password prompts may occur")];
-//	}
+	// This is how you add release notes and run specific migration steps
+	if (recordedVersionNumber < currentVersionNumber) {
```

Comment:
So now - this will be shown in every release iirc.

Reply:
yes .. proposed change is `(recordedVersionNumber <2061) {`

Reply:
SGTM